blaze/expr/arithmetic.py

```diff
@@ -378,7 +395,7 @@ class Not(UnaryOp):
     _dtype = ct.bool_
 
     def __str__(self):
-        return '~%s' % parenthesize(eval_str(self._child))
+        return '~%s' % parenthesize(eval_str(self._operands[0]))
```

Comment:
again, there's no need to change anything because you've already defined `self._child`